src/test/collectionHandlingFunctions.php

```diff
@@ -0,0 +1,314 @@
+<?php
+require "../collectionHandlingFunctions.php";
+use PHPUnit\Framework\TestCase;
+
+class collectionHandlingFunctions extends TestCase 
+{
+
+    public function testFailureDescribeCollectionItem_EmptyItem() 
```

Comment:
should be `_emptyItem` as opposed to `_EmptyItem` for consistency with the rest of the code

Reply:
Done!